### ISSUE TITLE
Only Video Decoder NetCast Legacy

### ISSUE THREAD

**Issue by ckorodenis** [mariotaku/moonlight-tv#465]
### Moonlight Version

1.6.22

### Device Type

webOS TV

### Device Model

LG 75UM7110

### Device OS

webOS ver: 4.10.0 - Firmware ver: 05.40.45

### Host OS

Windows

### Host Software

Sunshine

### Host Setup

i5-7400, GTX1060, Win11, Stream LAN 1Gb

### Bug Description

After the updates I stopped being able to use a decoder other than NetCast Legacy, if I select WebOS SMP steam doesn't turn on and only a black screen remains, Moonlight gets stuck, I have to turn it off. 

If I select WebOS NDL the settings don't change and the app itself selects NetCast Legacy. No other combinations work, I've tried everything, tried switching to Wi-Fi also without success, changed bitrate settings, minimum resolution, no positive results. The situation got worse after the update. 

Is the solution to use an older Moonlight? NetCast Legacy works, but after a few minutes the picture gets stuck and the stream stops. 

My goal is a stable FullHD stream, 60FPS over LAN connection. No 4K, no HDR, etc. so I don't know why it's so demanding on the connection ...

### Reproduction Steps

1. Normal start (Everything reinstalled, everything default, everything tested) 
2. Open Desktop
3. Waiting for connection => Black screen / stream stuck after 2~5 minutes

**Comment by mariotaku** [comment]
Hello, only NetCast Legacy is supported on your TV. I'll try to run it on a similar TV to see why the video stream stops.

**Comment by ckorodenis** [comment]
> Hello, only NetCast Legacy is supported on your TV. I'll try to run it on a similar TV to see why the video stream stops.

Hi! In that case my information was wrong, I thought the TV supports other video decoders. 

Thanks for the information, is it realistic to achieve FullHD 60FPS streaming using NetCast Legacy? Can I change the Sunshine settings to increase stability? 

What settings would you recommend in Moonlight? I want to test it with the setting you think will be most appropriate.

**Comment by mariotaku** [comment]
@ckorodenis Of course! The decoder should work with 1080p 60fps without issue. However, your GPU may not be powerful enough for streaming that while in games. IMO 2060 would be needed for smooth 1080p60 streaming.

**Comment by ckorodenis** [comment]
> @ckorodenis Of course! The decoder should work with 1080p 60fps without issue. However, your GPU may not be powerful enough for streaming that while in games. IMO 2060 would be needed for smooth 1080p60 streaming.

I can't stream lower resolutions on 4K TV anymore ...the upgrade is planned, I'm waiting for the right time 

Will limiting FPS to 30 be the solution? Everything got much worse after the update, I didn't have such problems before

**Comment by mariotaku** [comment]
I'm not sure what the exact situation is. I streamed 12 hours consecutively on UM7600 (having the same chip as yours) and didn't notice any issue.

Did you try to stream with your phone to see how it performs?

**Comment by ckorodenis** [comment]
On a mobile device, the stream works without difficulty. Unfortunately I can't get everything to work, after a few tests it's still the same, always the stream lasts stable for 5~8 minutes and then the image gets stuck, on PC the game continues to run without problems, I tried to change everything but without results (V-Sync Off-On, low graphics settings, same resolution everywhere, etc. 

Below screen of TV when stuck 

![IMG_0772](https://github.com/user-attachments/assets/983a906b-a14d-4ed5-bea7-3d9e86742b2c)

**Comment by mariotaku** [comment]
Streaming on k5lp should fully work with [proper jailer fix](https://gist.github.com/mariotaku/fab3ee34fae3415c8213d1a1639a6aff), since I'm not able to reproduce, I'll close this issue. Please try that fix and feel free to reopen if the issue still exist!
